VTEX - Search API.json

```diff
@@ -23440,31 +23442,35 @@
-                                                                "example": "Llf"
-                                                            },
-                                                            "nameComplete": {
-                                                                "type": "string",
-                                                                "description": "Complete product name.",
-                                                                "example": "Cacilds Llf"
-                                                            },
-                                                            "imageUrl": {
-                                                                "type": "string",
-                                                                "description": "Product image URL.",
-                                                                "example": "https://merch.vteximg.com.br/arquivos/ids/172754-25-25/Diablo.jpg?v=637521970448770000"
+                                                        "items":{
+                                                            "type": "object",
+                                                            "description": "Product information.",
+                                                            "properties": {
+                                                                "productId": {
+                                                                    "type": "string",
+                                                                    "description": "Product ID.",
+                                                                    "example": "1"
+                                                                },
+                                                                "itemId": {
+                                                                    "type": "string",
+                                                                    "description": "Item ID.",
+                                                                    "example": "3"
+                                                                },
+                                                                "name": {
+                                                                    "type": "string",
+                                                                    "description": "Product name.",
+                                                                    "example": "Llf"
+                                                                },
+                                                                "nameComplete": {
+                                                                    "type": "string",
+                                                                    "description": "Complete product name.",
+                                                                    "example": "Cacilds Llf"
```

Comment:
I think these examples could be better. Maybe we should get something from the docstore catalog?